core/miner_manager.go

```
@@ -385,3 +470,18 @@ func (mm *MinerManager) addGenesesMiners(miners []*types.Miner, accountDB types.
 		mm.addGenesisMinerStake(miner, accountDB)
 	}
 }
+
+func (mm *MinerManager) genGuardNodes(accountDB types.AccountDB) {
+	for _, addrStr := range types.ExtractGuardNodes {
+		addr := common.HexToAddress(addrStr)
+		miner := &types.Miner{ID: addr.Bytes(),Type:types.MinerTypeProposal,Identity:types.MinerGuard,Status:types.MinerStatusActive,ApplyHeight:0,Stake:0}
+		bs, err := msgpack.Marshal(miner)
+		if err != nil {
+			panic("encode miner failed")
+		}
+		initVoteInfo(accountDB,addr)
```

the error is ignored